packages/atomic/src/components/common/result-list/result-list-common.tsx

```diff
@@ -67,6 +67,30 @@ export class ResultListCommon<SpecificResult extends AnyResult = AnyResult>
     this.props.nextNewResultTarget.focusOnNextTarget();
   }
 
+  public get quickviews() {
```

Comment:
I'm interested in hearing what Jean Felix's opinion first about this first.
If that's the consensus I will change it in a separate PR and implement Quickview pagination directly in Headless.

The main reason I did it here is that it seems to be really a UI choice/concern to manage navigating between preview of different results in a result list.

For this PR though, I would leave it as is.

Reply:
I agree with Benjamin, it seems a better fit for Headless, even if it stems from a UI requirement, in the end it handles state, and it could be used by many UIs

Reply:
Ok !
https://coveord.atlassian.net/browse/KIT-2205